util/src/main/scala/Money.scala

```diff
@@ -13,8 +13,13 @@ import scala.math._
 import BigDecimal.RoundingMode._
 import scala.math.BigDecimal.RoundingMode
 import ValidatedFlatMapFeature._
+import io.sphere.util.BaseMoney.bigDecimalToMoneyLong
 import io.sphere.util.Money.ImplicitsDecimal.MoneyNotation
 
+class MoneyOverflowException(amount: BigDecimal)
+    extends RuntimeException(
+      s"Money operation resulted in an overflow. $amount is too big or small.")
```

Comment:
We don't really need the `amount` here, but it could be helpful for debugging.

Reply:
Is it possible to send a very very long big decimal, that would result in a very long exception message, overflowing our log systems?

Reply:
It's theoretically possible. In practice, right now, it's not, because all of our Money-related JSON and GraphQL decoders only read `Long` values. I'll remove it anyways, to protect against possible future changes.

Reply:
done